dvc/checkout.py

```diff
@@ -64,7 +60,12 @@ def _changed(path_info, tree, hash_info, cache, filter_info=None):
         logger.debug("cache for '%s'('%s') has changed.", path, hi)
         return True
 
-    actual = tree.get_hash(path)
+    try:
+        actual = tree.get_hash(path)
```

Comment:
Some clouds don't properly wrap exceptions into OSErrors, but the followup PR will move `get_hash` out of trees into a separate helper and those clouds will use stat() that uses ls() that handles exceptions properly.